### DISCUSSION TITLE
add support for hiding jobs from specific locations

### DISCUSSION THREAD

**Discussion by FSou1** [damianmgarcia/Hide-n-Seek#10] [Ideas]
Hi, first of all, the extension works great, thank you, I really enjoy using it. I wonder if the extension can support hiding jobs from specific cities and states.

**Comment by damianmgarcia** [comment]
Thank you! I am so happy that you are enjoying it. It means a lot to me.

Your idea is a good one. 

I need some time to think about it, though. Additional hideable attributes beyond the current two (company name & promoted) would entail designing a more scalable UI, which I don't mind doing, but would take some time.

There are currently two requests for additional hideable attributes (this one and #9), and there may be more requests in the future, so a potential update with a more scalable UI that would better accommodate additional hideable attributes is definitely on the radar.

**Comment by damianmgarcia** [comment]
Keyword search was added a few weeks ago which should cover hiding cities, states, and other locations. I know it's been a while, but thanks again for the suggestion! ' v '
